tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 skipsdist = True
 envlist =
-  py{35,36}-dj{1.11,2.0}
+  py{35,36}-dj{1.11,2.0, 2.1}
```

Comment:
Does this space here work correctly?

Reply:
Yes it works. But I have a problem with tests related to django 1.11. I get the following error:

> ModuleNotFoundError: No module named 'tests 

And also, when I want to run the tests, I will add Postgres password to my database settings. I don't know how your tests pass without setting it.

Reply:
Let's keep it consistent and remove it I think. Tox is a dark art sometimes.

I run them against a Docker postgres container. Not sure why 1.11 is failing for you there. It is passing in CI.